<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectEngineerWidget</class>
 <widget class="QWidget" name="SelectEngineerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myqrc.qrc">
    <normaloff>:/myqrc/Technical.png</normaloff>:/myqrc/Technical.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="queryCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>条件查询</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="queryComboBox">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="queryLineEdit">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="queryBt">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="detailCheckBox">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>详查</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ModifyCheckBox">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>修改</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="EngineerListView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="myqrc.qrc"/>
 </resources>
 <connections/>
</ui>
